src/main/java/randoop/main/Minimize.java

```diff
@@ -160,30 +159,35 @@ public boolean handle(String[] args) throws RandoopTextuiException {
       }
     } catch (Options.ArgException ae) {
       usage("while parsing command-line arguments: %s", ae.getMessage());
+      throw new RandoopTextuiException("See usage.");
     }
 
     if (Minimize.suitepath == null) {
-      System.out.println("You must specify an input file path.");
-      System.out.println("Use the --suitepath option.");
-      System.exit(1);
+      throw new RandoopTextuiException(
+          "You must specify an input file path to the 'minimize' command. Use the --suitepath option.");
     }
 
     // Check that the input file is a Java file.
     if (!FilenameUtils.getExtension(Minimize.suitepath).equals("java")) {
-      System.out.println("The input file must be a Java file.");
-      System.exit(1);
+      throw new RandoopTextuiException("The input file must be a Java file.");
     }
 
     if (Minimize.testsuitetimeout <= 0) {
-      System.out.println("You must specify a positive timeout value.");
-      System.exit(1);
+      throw new RandoopTextuiException("You must specify a positive timeout value.");
     }
 
     // File object pointing to the file to be minimized.
     File originalFile = new File(suitepath);
 
     // Call the main minimize method.
-    return mainMinimize(originalFile, suiteclasspath, testsuitetimeout, verboseminimizer);
+    boolean success = false;
+    try {
+      success = mainMinimize(originalFile, suiteclasspath, testsuitetimeout, verboseminimizer);
+    } catch (IOException e) {
+      System.err.println("IOException: " + e.getMessage());
```

Comment:
Simply printing an error will cause it to be overlooked, and users may be confused by the result.  I think this should throw an exception instead.